policies/lib/tree/30_generic_methods/file_augeas.cf

```diff
@@ -0,0 +1,87 @@
+#####################################################################################
+# Copyright 2025 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+# @name File augeas
+# @description Manage files using Augeas
+# @documentation See TODO.
+# @parameter            path Path of the file
+# @parameter            script Commands to run on the file
+# @parameter            if_script Only run the script if this condition is true
```

Comment:
Do not use "condition" here as it is already used by the technique editor logic.
Maybe "Condition the script execution only if the if_script succeeds"